projects/observability/src/shared/components/cartesian/cartesian-chart.component.scss

```diff
@@ -104,6 +104,10 @@
     }
 
     .legend-entries {
+      /* Prevent large legends from shrinking chart */
+      max-height: 3.5rem;
+      overflow: auto;
```

Comment:
I am guessing it is only for vertical scroll, if yes then we can do `overflow-y: auto` instead.

Reply:
Makes sense. Done. ✅